credentials.py

```diff
@@ -58,9 +58,8 @@ def verify_jwt(credentials: Credentials) -> bool:
 
 
 async def get_token_from_bearer(
-    request: Request,
```

Comment:
The purpose of this was just to demonstrate that `request` is available in the dependable function. We could change to an unnamed var? E.g. `_: Request`